### PR TITLE
docs: required information for OAuth2 configuration

### PR DESCRIPTION
**Pull request by shawnzhu**
### SUMMARY

Adding required steps when configuring OAuth2 authentication.

### ADDITIONAL INFORMATION

- [x] Has associated issue: #12855
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: docs/src/pages/docs/installation/configuring.mdx [hunk 1]
```diff
@@ -113,6 +113,8 @@ RequestHeader set X-Forwarded-Proto "https"
 Beyond FAB supported providers (Github, Twitter, LinkedIn, Google, Azure, etc), its easy to connect
 Superset with other OAuth2 Authorization Server implementations that support “code” authorization.
 
+Make sure the pip package `Authlib` is installed on the webserver.
```
Comment (nytai) [inline docs/src/pages/docs/installation/configuring.mdx:116]: Can we link to the pip package?

**Review by nytai — APPROVED** [review]
LGTM. I left an optional comment. LMK if you want to address or merge as is

File: docs/src/pages/docs/installation/configuring.mdx (hunk 1 above)
Comment (shawnzhu) [reply docs/src/pages/docs/installation/configuring.mdx]: addressed in https://github.com/apache/superset/pull/15010/commits/bb2253e5a541af60b8777519f401837e27310a22

**Comment by shawnzhu** [comment]
@nytai I've added the link to the pip package `authlib`

**Review by nytai — APPROVED** [review]

**Pull request merged**